editoast/osm_to_railjson/README.md

```diff
@@ -0,0 +1,17 @@
+# Converter from OSM to raijson
+
+## Usage
+
+Example for Germany:
+
+1. Download OSM's German data (~4Gb `germany-latest.osm.pbf`) from
+    https://download.geofabrik.de/europe/germany.html
+2. Launch conversion (release build of editoast and conversion can be long):
+    ```sh
+    cd ../../editoast
+    cargo run -r -- osm-to-railjson <path/to/germany-latest.osm.pbf> <path/to/germany_railjson.json>
```

Comment:
nit: please always prefer long options, the code is usually more read than written.
```suggestion
    cargo run --release -- osm-to-railjson <path/to/germany-latest.osm.pbf> <path/to/germany_railjson.json>
```

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/8825/commits/d7b9271eab6555228d00ef4c323cfb035ba06439